network/manager.go

```diff
@@ -807,6 +810,25 @@ func (nm *networkManager) GetEndpointInfosFromContainerID(containerID string) []
 	return ret
 }
 
+// Get PnP Device ID
+func (nm *networkManager) GetPnPDeviceID(instanceID string) (string, error) {
```

Comment:
just wanted to confirm that the instanceID isn't controlled by customer or anything right? like it's not possible to execute arbitrary powershell commands with this

Reply:
is this and the other powershell command functions unit-testable?

Reply:
instanceID is from CNS as PnPID/PciID